bw2data/project.py

```diff
@@ -35,6 +39,10 @@ def lockable():
 
 
 class ProjectDataset(Model):
+    # Event sourcing
+    is_sourced = BooleanField(default=False, constraints=[SQL("DEFAULT 0")])
```

Comment:
[Peewee schema migrations](https://docs.peewee-orm.com/en/latest/peewee/playhouse.html#schema-migrations).

Reply:
Cool, I'll add this in one of the next commits, cheers.